doc/release-notes/rl-2511.section.md

```diff
@@ -175,6 +175,10 @@
   NEWS can be viewed from Emacs by typing `C-h n`, or by clicking `Help->Emacs News` from the menu bar.
   It can also be browsed [online](https://git.savannah.gnu.org/cgit/emacs.git/tree/etc/NEWS?h=emacs-30).
 
+- `heif-image-plugin` has been dropped due to lack of upstream maintenance and breakage. Use `pillow-heif` instead.
+
+- `pyheif` has been dropped due to lack of upstream maintenance and breakage. Use `pillow-heif` instead.
```

Comment:
```suggestion
- `python3Packages.heif-image-plugin` has been dropped due to lack of upstream maintenance and breakage. Use `pillow-heif` instead.

- `python3Packages.pyheif` has been dropped due to lack of upstream maintenance and breakage. Use `pillow-heif` instead.
```
but I don't think these are needed given the lines in `python-aliases.nix`

Reply:
fair, let me fix that